docs/src/plasma/plasma-cnc-primer.adoc

```diff
@@ -519,11 +519,11 @@ Because the full scale is 10 Volts, then the frequency per Volt is:
 
 So assuming we have a 5 Volt input, the calculated frequency would be:
 
-(2520 Hz/V * 5 V) + 3,800 Hz = 16,400 Hz
+(2,520 Hz/V * 5 V) + 3,800 Hz = 16,400 Hz
```

Comment:
```suggestion
(2520 Hz/V * 5 V) + 3800 Hz = 16,400 Hz
```
IMHO these thousand separators are more confusing than helpful for such small numbers...